galaxy/worker/importers/base.py

```diff
@@ -50,8 +50,10 @@ def make_content(self):
         repo = self.ctx.repository
         ns = repo.provider_namespace.namespace
 
-        name = repo.name
-        original_name = repo.name
+        # Name is the content name, which in the case of multi content repos
+        # might not be the same as the repository name
+        name = repo.original_name
```

Comment:
Name should default to the repo.name, which should have been sanitized of any '-' and '.' chars and converted to lowercase() somewhere.